pom.xml

```diff
@@ -994,6 +1506,284 @@
             <artifactId>gson</artifactId>
             <version>${gson.version}</version>
         </dependency>
+        
+        <dependency>
+            <groupId>org.kuali.rice</groupId>
+            <artifactId>rice-core-impl</artifactId>
```

Comment:
This PR is explicitly adding in some of the Rice dependencies and then making Log4J adjustments to those, rather than making the adjustments to only the existing Rice dependencies. Can these extras be removed so that the exclusions can be kept to the existing imports?

If you do need to keep them, then make sure they have the other exclusions that the existing imports have (xapool, etc.).

Reply:
thanks for noticing, I added these manually, they are not needed, I shall remove these entries